openprescribing/pipeline/management/commands/delete_latest_prescribing_import.py

```diff
@@ -0,0 +1,166 @@
+"""
+This command:
+
+    * deletes all prescribing data from:
+      * the filesystem
+      * BigQuery
+      * Cloud Storage
+      * the matrixstore
```

Comment:
I'm not sure it's quite right to say "the matrixstore" here. It certainly took me a bit by surprise and then I had to confirm by looking at the code that it didn't actually do this. I think it might be more accurate to say:
```suggestion
    * deletes all prescribing data (both original data and extracts created by the matrixstore build) from:
      * the filesystem
      * BigQuery
      * Cloud Storage
```